topics/modules-api-ref.md

```diff
@@ -339,7 +339,7 @@ example "write deny-oom". The set of flags are:
                    Starting from Redis OSS 7.0 this flag has been deprecated.
                    Declaring a command as "random" can be done using
                    command tips, see https://valkey.io/topics/command-tips.
-* **"allow-stale"**: The command is allowed to run on slaves that don't
+* **"allow-stale"**: The command is allowed to run on replicas that don't
```

Comment:
This file is generated from the various comment blocks in https://github.com/valkey-io/valkey/blob/unstable/src/module.c, so can you also submit a PR to update this file in the main repo as well?